CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 2.6.0-beta3
 - [Ignore Deprecated events if running under netcore20](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/848)
 - [Implement unhandled exception auto-tracking (500 requests) for MVC 5 and WebAPI 2 applications.](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/847)
+- [Enable .NET Core platform in WindowsServer SDK, completing heartbeat feature across platforms](https://github.com/Microsoft/ApplicationInsights-dotnet-server/issues/854)
```

Comment:
Can you add more details to the issue? Specifically what Initializers/Modules from WindowsServer package are made available for .net core?

Reply:
Done, I enumerated all the modules that are now enabled in .NET Standard applications.